routes/lesson.js

```diff
@@ -96,6 +97,32 @@ router.post("/lesson", async (request, response) => {
     });
 });
 
+/*
+ * @api [post] /chapter
+ *  summary: "Post Surah Informations"
+ *  tags:
+ *    - Chapter Endpoints
+ *  produces:
+ *    - application/json
+ *  parameters:
+ *        - in: body
+ *          name: id
+ *          description: the lesson to update and it's new attributes
+ *          schema:
+ *              $ref: '#/definitions/Lesson'
+ *  responses:
+ *    200:
+ *      description: Chapter has been created.
+ *
+ */
+router.post("/chapter", async (request, response) => {
+    await lesson.createLesson(request.body).then(async function (result) {
```

Comment:
So, here is where you want to change the function you are calling. I suggest creating the model function inside this file for now and figuring the import stuff later on.